provider/gce/google/conn_network.go

```diff
@@ -216,6 +216,11 @@ func (gce Connection) ClosePorts(target string, rules corefirewall.IngressRules)
 	return nil
 }
 
+// RemoveFirewall removed the named firewall from the project.
```

Comment:
`removes`.

Reply:
Haha, I copied and pasted this. Nice work Harry.